content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,41 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+And add a new stage to the pipeline:
 
 ```dvc
-$ dvc repro --downstream
+$ dvc run -n final -d count.txt -o alphabet.txt \
+            "cat count.txt | egrep -o '[a-zA-Z]+' > alphabet.txt"
```

Comment:
Let's try to change this last stage a little to something closer to real life. It needs a meaningful name and some new command. It probably shouldn't use count.txt as a dependency too because that's an untracked metric file (-M option) so how about using the numbers.txt file as dependency here? It doesn't matter that it was already used for `count`. Just process it in some other hypothetical way similar to how `count` has a hypothetical `process.py` Python script.

Reply:
ACTUALLY, this whole new stage may not be needed at all. Please follow the discussion above with Ivan and change accordingly. See https://github.com/iterative/dvc.org/pull/1624#pullrequestreview-455125609

Reply:
> ACTUALLY, this whole new stage may not be needed at all.

Yeah you are right. Updated the example.

Reply:
Thanks